packages/vuetify/src/components/VAlert/__tests__/VAlert.spec.ts

```diff
@@ -60,7 +63,7 @@ describe('VAlert.ts', () => {
     })
 
     const icon = wrapper.find('.v-alert__dismissible')
-    const input = jest.fn(show => wrapper.setProps({ show }))
```

Comment:
looks like this wasn't doing anything, removing it made no difference

Reply:
I think that was supposed to be like v-model from before it stored its own state.